db/schema.rb

```diff
@@ -75,15 +75,13 @@
     t.integer  "lesson_id"
     t.string   "text"
     t.string   "cited_src"
-    t.boolean  "approved",              default: true
-    t.integer  "twitter_id",  limit: 8
+    t.boolean  "approved",             default: true
+    t.integer  "twitter_id", limit: 8
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "tweet_index",                          null: false
```

Comment:
Update: I talked to @TheJhyde in person and he clarified that a later commit undid this change.  The schema still includes a tweet_index for each tweet. 👍